python/tests/compat/test_mantid.py

```diff
@@ -300,7 +299,6 @@ def test_EventWorkspace_with_monitors(self):
             # Absence of the following is not crucial, but currently there is
             # no need for these, and it avoid duplication:
             assert 'detector-info' not in monitor.coords
-            assert 'run' not in monitor.coords
```

Comment:
Your PR message mentions that run info is not stored for monitors? Should this line actually be removed then?

Reply:
There is never a coord called "run" created in the data or monitor workspace unless there happens to be a run log called "run" in the workspace. I don't see a good reason for explicitly requiring that there isn't a coord called "run".

Reply:
Ah, good point, the test made sense only in the old context... but maybe we should now check instead that the logsare not duplicated into the monitor? This might happen because we are using `ExtractMonitors` to split the monitors off the data workspace.